serialization/bson.ksy

```diff
@@ -140,23 +140,23 @@ types:
         type:
           switch-on: type_byte
           cases:
-            'bson_type::number_double': f8
-            'bson_type::string': string
-            'bson_type::document': bson
-            'bson_type::array': bson
-            'bson_type::bin_data': bin_data
-            'bson_type::object_id': object_id
-            'bson_type::boolean': u1
-            'bson_type::utc_datetime': s8 #The int64 is UTC milliseconds since the Unix epoch.
-            'bson_type::reg_ex': reg_ex
-            'bson_type::db_pointer': db_pointer
-            'bson_type::javascript': string
-            'bson_type::symbol': string # a programming language (e.g., Python) symbol
-            'bson_type::code_with_scope': code_with_scope
-            'bson_type::number_int': s4
-            'bson_type::timestamp': timestamp
-            'bson_type::number_long': s8
-            'bson_type::number_decimal': f16
+            bson_type::number_double: f8
+            bson_type::string: string
+            bson_type::document: bson
+            bson_type::array: bson
+            bson_type::bin_data: bin_data
+            bson_type::object_id: object_id
+            bson_type::boolean: u1
+            bson_type::utc_datetime: s8   #The int64 is UTC milliseconds since the Unix epoch.
+            bson_type::reg_ex: reg_ex
+            bson_type::db_pointer: db_pointer
+            bson_type::javascript: string
+            bson_type::symbol: string   # a programming language (e.g., Python) symbol
```

Comment:
Why do you introduce triple spaces here?

Let's follow this rule for KSY comments - exactly one space before the number sign (`#`), one space after it and then the actual message.

```suggestion
            bson_type::utc_datetime: s8 # The int64 is UTC milliseconds since the Unix epoch.
            bson_type::reg_ex: reg_ex
            bson_type::db_pointer: db_pointer
            bson_type::javascript: string
            bson_type::symbol: string # a programming language (e.g., Python) symbol
```